extensions/cms/src/apiWrapper.ts

```diff
@@ -156,7 +158,7 @@ export class ApiWrapper {
 	}
 
 	public showWarningMessage(message: string, ...items: string[]): Thenable<string | undefined> {
-		return vscode.window.showWarningMessage(message, ...items);
+		return vscode.window.showWarningMessage(message, { modal: true }, ...items);
 	}
```

Comment:
Why is this one the only modal one?

Reply:
(while I understand why you want this to be modal I'm more suggesting you add this as a param to the method. Sure it may only ever be passed as true currently but at least it shows intent and if future devs add calls to this they won't be surprised when the dialog is mysteriously modal)

---
In reply to: [289261130](https://github.com/microsoft/azuredatastudio/pull/5542#discussion_r289261130) [](ancestors = 289261130)